addons/crm/views/crm_lead_views.xml

```diff
@@ -996,6 +996,7 @@
                     <separator/>
                     <filter string="Won" name="won" domain="['&amp;', ('active', '=', True), ('stage_id.is_won', '=', True)]"/>
                     <filter string="Lost" name="lost" domain="['&amp;', ('active', '=', False), ('probability', '=', 0)]"/>
+                    <filter string="Ongoing" name="ongoing" domain="['&amp;', ('active', '=', True), ('stage_id.is_won', '=', False)]"/>
```

Comment:
Hi @vivp-odoo :slightly_smiling_face: 

Could you set `filter_won`, `filter_ongoing` and `filter_lost` for the names? :slightly_smiling_face: 
I know it isn't done here but we tend to set `filter_%`  for the filter tags names.

Also, if I look at the spec image, I think the ongoing filter needs to be between the won and the lost one :thinking: 
A small rebase could also be great :smile: 

Thank you!